plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1270,11 +1270,24 @@ public static function get_subscriptions() {
 			return $data;
 		}
 
+		$source = $_SERVER['REQUEST_URI'];
```

Comment:
I would name this `$url` or suchlike, and initialise an empty string `$source`, to make it clear that `$source` is something else.

Reply:
That makes sense. I went with `$request_uri`, as that's ultimately what it is.